code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -631,6 +631,10 @@
 
 /obj/item/clothing/shoes/drip/dropped(mob/user)
 	. = ..()
```

Comment:
you should move the . = ..() call as it moves the item out

Reply:
not sure if it will cause an issue tho

Reply:
method to fix the issue was copied directly from krav maga gloves, I don't think it causes issues.